src/apof/portal/static/portal/css/topbar.css

```diff
@@ -1,55 +1,67 @@
 #top-nav{
+    width: 100%;
     padding-top: 9px;
-    opacity: 0.95;
+
+    -webkit-transition: width .5s ease-in-out;
+       -moz-transition: width .5s ease-in-out;
+         -o-transition: width .5s ease-in-out;
+            transition: width .5s ease-in-out;
```

Comment:
It seems too me that your argument is invalid because most if not all modern editors have functionality to edit multi-lines.